GERTi/GERTiMNC.lua

```diff
@@ -230,7 +262,7 @@ handler["RegisterNode"] = function (eventName, receivingModem, sendingModem, por
 			subChildDex = subChildDex + 1
 		end
 	end
-	transmitInformation(sendingModem, port, childNodes[nodeDex]["gAddress"])
+	transmitInformation(sendingModem, port, "RegisterComplete", originatorAddress, childNodes[nodeDex]["gAddress"])
```

Comment:
RegisterComplete is no longer used, so these additional arguments are no longer necessary. Please remove them.